### PR TITLE
drop unused deps, add simple integration test, change typescript target

### PR DESCRIPTION
**Pull request by christianhaller3000**
Hi Caleb, 
the generated polyfill includes the optional chaining operator `?.`
Your `.babelrc` targets IE11, but I think it is not used in your rollup process.

That's why I dropped babel und added es2015 to the typescript target.
A simple integration test ensures the generated output is free from fancy syntax.

### CODE REVIEW

File: jsdom-tests/jsdom.test.js [hunk 1]
```diff
@@ -5,12 +5,14 @@ const polyfillContents = readFileSync('./dist/index.js', 'utf-8');
 
 function test(title, condition) {
   if (!condition) {
-    throw new Error(`${title} failed with error ${error}`);
+    throw new Error(`${title} failed with error`);
```
Comment (christianhaller3000) [inline jsdom-tests/jsdom.test.js:8]: error is undefined

File: package.json [hunk 2]
```diff
@@ -72,14 +68,12 @@
...
     "lit-html": "^1.2.1",
     "rollup": "^2.46.0",
-    "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-cleanup": "^3.2.1",
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-livereload": "^1.2.0",
     "rollup-plugin-serve": "^1.0.1",
     "sinon": "^9.2.4",
     "standard-version": "^9.0.0",
-    "tsc": "^1.20150623.0",
```
Comment (christianhaller3000) [inline package.json:82]: tsc comes with typescript

File: rollup.config.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import livereload from 'rollup-plugin-livereload';
 import cleanup from 'rollup-plugin-cleanup';
 
 const plugins = [typescript({
+  target: 'es2015',
```
Comment (christianhaller3000) [inline rollup.config.js:7]: that is the important line 🔥

**Review by calebdwilliams — APPROVED** [review]
lgtm

**Pull request merged**